components/net/http_loader.rs

```diff
@@ -1564,9 +1564,10 @@ fn http_network_fetch(
 
     // Substep 2
 
-    // TODO Determine if response was retrieved over HTTPS
-    // TODO Servo needs to decide what ciphers are to be treated as "deprecated"
```

Comment:
(I removed this comment because deprecated ciphers have been removed from Servo to align with the deprecation process of other browsers.)

Reply:
👍